app/views/documents/_attachment.html.erb

```diff
@@ -56,6 +56,11 @@
         <%= attachment_attributes(attachment) %>
       <% end %>
     </p>
+    <% if attachment.is_official_document? %>
```

Comment:
Should this have a test?

Reply:
I'm happy that this is tested further down the line which is why i've not written one but will write one if you think it needs it :)